container/dockershell.sh

```diff
@@ -145,6 +150,11 @@ if [ -z ${DEPLOYMENT_TYPE} ]; then
     DEPLOYMENT_TYPE=$defaultDeployType
 fi
 
+if [ -z ${ES_VERSION} ]; then
+    printf "Event Store version (--es-version) not specified, please provide"
+    DEPLOYMENT_TYPE=$defaultDeployType
```

Comment:
hmm is this line necessary here ? how is this related to the ES_VERSION ?

Reply:
+1 I think this was a typo / copy and paste error.

Reply:
yes, that if test is proposed to be removed because version existance tested for in setup container